README.md

```diff
@@ -23,23 +23,94 @@ Explore [**documentation**](https://kotlin.github.io/dataframe/overview.html) fo
 
 ## Setup
 
-### Gradle
-```kotlin
+### Gradle for Server-side
```

Comment:
I would call it "Gradle for JVM". It's not like DataFrame is just for servers. Many people use it on their desktop.